File: 241026/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    // 여기에 코드를 작성해주세요.
    string arr[5] = {"apple", "banana", "grape", "blueberry", "orange"};
    char alp;
    cin >> alp;
    int cnt = 0;

    for(int i = 0; i < 5; i++){
        for(int j = 2; j < 4; j++){
            if(alp == arr[i][j]){
                cout << arr[i] << endl;
                cnt++;
                break;
            }
        }
    }

    cout << cnt;

    return 0;
}

File: javascript/trail0/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.js
const fs = require("fs");
let input = fs.readFileSync(0).toString().trim();

const fruits = ["apple", "banana", "grape", "blueberry", "orange"];

let count = 0;

for (let i = 0; i < fruits.length; i++) {
    if (fruits[i][2] === input || fruits[i][3] === input) {
        console.log(fruits[i]);
        count++;
    }
}

console.log(count);